src/rqt_image_view/image_view.cpp

```diff
@@ -375,6 +385,32 @@ void ImageView::onHideToolbarChanged(bool hide)
   ui_.toolbar_widget->setVisible(!hide);
 }
 
+void ImageView::invertPixels(int &x, int &y)
+{
+  for (int i=0; i<3; i++)
+  {
+  	// Could just do 255-conversion_mat_.at<cv::Vec3b>(cv::Point(x,y))[i], but that doesn't work well on gray
+    if ( conversion_mat_.at<cv::Vec3b>(cv::Point(x,y))[0]>127 )
+      conversion_mat_.at<cv::Vec3b>(cv::Point(x,y))[i] = 0;
+    else
+      conversion_mat_.at<cv::Vec3b>(cv::Point(x,y))[i] = 255;
+  }
+}
+
+void ImageView::overlayGrid()
+{
+  getNodeHandle().param("/rqt_image_view/num_gridlines", num_gridlines_, 4);
```

Comment:
How will this interact with multiple image view plugins open? I guess it wouldn't be possible to use different settings for them. Would it be better to configure this in the plugin instead and persist the value in the settings?